### DISCUSSION TITLE
what library mpv uses for subtitle rendering?

### DISCUSSION THREAD

**Discussion by rezad1393** [mpv-player/mpv#14640] [Q&A] [answered]
I have an issue with Right to left languages (Persian) and I wan to report it.
where I can report it?

**Comment by avih** [comment]
https://github.com/libass/libass

**Reply by rezad1393** [reply]
I should report it there corrent?
does vlc use the same library?
because I think I can reproduce the issue there too.

**Reply by avih** [reply]
> I should report it there corrent?

Maybe, but you never said what the issue is, so it's hard to tell whether it's an mpv or libass issue.

> does vlc use the same library?

Not sure, and it's complex, because even if it doesn't use libass then it uses a different library which libass tries to immitate.

Just report it either in mpv or at libass, whichever you think is more appropriate, and hopefully if you chose wrongly then people will tell you.

**Reply by rezad1393** [reply]
cool
the issue is that when subitle is right to left language and a line start with a number then the number (incorrectly) is shown at the end of line.

**Reply by avih** [reply]
Just report it properly please, at the issue tracker and not as a discussion.

**Reply by rezad1393** [reply]
report it here or at libass

**Reply by avih** [reply, marked as answer]
> Just report it either in mpv or at libass, whichever you think is more appropriate, and hopefully if you chose wrongly then people will tell you.

**Reply by moi15moi** [reply]
There seems to be already a issue opened about that on libass repos: https://github.com/libass/libass/issues/538
